build/Dockerfile

```diff
@@ -15,13 +15,20 @@ RUN \
  iptables \
  nodejs \
  parallel \
+ unzip \
  && \
  apt-get clean autoclean && \
  apt-get autoremove -y && \
  git clone --depth 1 https://chromium.googlesource.com/chromium/src/tools/clang && \
  clang/scripts/update.py && \
  rm -rf /tmp/*
 
+RUN \
+  cd /usr/local/bin && \
+  curl -O -J --silent https://releases.hashicorp.com/terraform/0.7.3/terraform_0.7.3_linux_amd64.zip && \
```

Comment:
Seems fine.

On Tue, Sep 13, 2016 at 10:25 AM, Jordan Lewis <notifications@github.com>
wrote:

> In build/Dockerfile
> <https://github.com/cockroachdb/cockroach/pull/9301#discussion_r78569960>:
>
> >   && \
> >   apt-get clean autoclean && \
> >   apt-get autoremove -y && \
> >   git clone --depth 1 https://chromium.googlesource.com/chromium/src/tools/clang && \
> >   clang/scripts/update.py && \
> >   rm -rf /tmp/*
> >
> > +RUN \
> > +  cd /usr/local/bin && \
> > +  curl -O -J --silent https://releases.hashicorp.com/terraform/0.7.3/terraform_0.7.3_linux_amd64.zip && \
>
> It lets curl set the filename by the http header, which i guess isn't
> necessary in this situation eh? I've removed this but am not going to
> rebuild the image with the change.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/9301/files/8d819cac420f15efa46db78ab93fc7da599b802f#r78569960>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPAhsKa34u1_e-09Y6npFehewGvWrks5qprJxgaJpZM4J660t>
> .
>